internal/fakecgo/go_libinit.go

```diff
@@ -20,9 +20,12 @@ func x_cgo_notify_runtime_init_done() {
 //
 //go:nosplit
 func _cgo_try_pthread_create(thread *pthread_t, attr *pthread_attr_t, pfn unsafe.Pointer, arg *ThreadStart) int {
-	var tries int
+	var ts syscall.Timespec
+	// tries needs to be the same type as syscall.Timespec.Nsec
+	// but the fields are int32 on 32bit and int64 on 64bit
+	// we assign tries to whatever type syscall.Timespec.Nsec is
```

Comment:
I don't understand this sentence...

```go
	// tries needs to be the same type as syscall.Timespec.Nsec
	// but the fields are int32 on 32bit and int64 on 64bit.
	// We assign to tries whatever type syscall.Timespec.Nsec is
```

?

Reply:
It's just describing why it's tries = ts.Nsec. It's because the type is different on each architecture

Reply:
Yeah, I know the meaning but the sentence seemed tricky (disclaimer: I'm not an English native speaker)

Reply:
So my question is:

1. Is the setence ended at `64bit`?
2. Isn't `assign to tries` correct rather than `assign tries to`?

Reply:
The sentence does end after 64Bit. I believe the second sentence is right. It might sound weird bc `tries` is being used as a noun instead of a verb in this case.

Reply:
At least, `We` is better than `we`

Reply:
So, the second sentence means

```
We assign `tries` to something whose type is type syscall.Timespec.Nsec
```

?

Reply:
Basically.

Perhaps it's better phrased as `We assign `tries` to syscall.Timespec.Nsec in order to match its type.`

?

Reply:
Yeah I think it's much better

In addition, avoiding `we` is nicer if possible, since it is not clear who is `we`, but this is optional. (We would be able to avoid `we` by using a passive sentence like "`tries` is assigned to ...")